chromium_src/chrome/renderer/extensions/chrome_extensions_dispatcher_delegate.cc

```diff
@@ -213,8 +213,8 @@ void ChromeExtensionsDispatcherDelegate::PopulateSourceMap(
                              IDR_ATOM_PROTOCOL_BINDINGS_JS);
   source_map->RegisterSource("ChromeSetting", IDR_CHROME_SETTING_JS);
   source_map->RegisterSource("ContentSetting", IDR_CONTENT_SETTING_JS);
-  source_map->RegisterSource("ChromeDirectSetting",
-                             IDR_CHROME_DIRECT_SETTING_JS);
+  source_map->RegisterSource("EasyUnlockProximityRequired",
```

Comment:
We actually needed direct setting for something completely different so this can probably be left out